cogs5e/sheetManager.py

```diff
@@ -373,7 +373,7 @@ async def character(self, ctx, *, name: str = None):
 
         if name is None:
             active_character: Character = await Character.from_ctx(ctx)
-            return await ctx.send(f'Currently active: {active_character.name}')
+            return await ctx.send(f'Currently active: {active_character.name} ({", ".join(active_character.active_context(ctx))}, `{active_character.sheet_type}`)')
```

Comment:
I think it's worth rewriting this to be an embed, such as
<img width="530" alt="Screenshot 2021-09-20 at 12 19 33" src="https://user-images.githubusercontent.com/8866981/134061545-66cfab7c-b15d-48a2-98e6-3cebba3a473b.png">